README.md

```diff
@@ -66,10 +66,10 @@ You can pass an options argument:
 
 ```js
 const s = new MagicString(someCode, {
-  // both these options will be used if you later
-  // call `bundle.addSource( s )` - see below
+  // these options will be used if you later call `bundle.addSource( s )` - see below
   filename: 'foo.js',
-  indentExclusionRanges: [/*...*/]
+  indentExclusionRanges: [/*...*/],
+	ignoreList: true
```

Comment:
```suggestion
  // market source as ignore in DevTools, see below #Bundling
	ignoreList: false
```

I think it's better to list `false` here to indicate it's default value

Reply:
Done.